kernel/src/analysis/javascript.rs

```diff
@@ -418,10 +419,12 @@ res
                     rule.name, filename, e
                 );
             }
-            let error_message = match e.to_string().find("at rule_code") {
-                Some(pos) => e.to_string()[..pos].to_string(),
-                None => e.to_string(),
-            };
+
+            let err_str = e.to_string();
+            let error_message = err_str
```

Comment:
Readability and conciseness change. See this [clippy lint](https://rust-lang.github.io/rust-clippy/master/index.html#/option_if_let_else).